src/main/resources/lib/azure-ad-id-provider/auth/group.js

```diff
@@ -148,7 +148,7 @@ function fromGraph(params) {
     // https://developer.microsoft.com/en-us/graph/graph-explorer?request=me/memberOf&method=GET&version=v1.0&GraphUrl=https://graph.microsoft.com
     var groupRequest = {
         method: 'GET',
-        url: 'https://graph.microsoft.com/v1.0/users/' + params.jwt.payload.oid + '/memberOf',
+        url: 'https://graph.microsoft.com/v1.0/users/' + params.jwt.payload.oid + '/memberOf?$top=' + idProviderConfig.pageSize,
```

Comment:
If the config of the idprovider has not been changed after upgrading, `idProviderConfig.pageSize` would still be `undefined`. The `<default>` value would not be applied unless you edit and safe the idprovider config.

Would it not be prudent to add a fallback value here too?

Reply:
Updated this part.